core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -313,6 +314,45 @@ private SqlCall makeSubstringSqlCall(SqlCall call) {
     return new SqlBasicCall(SUBSTR, sqlNodes, SqlParserPos.ZERO);
   }
 
+
+  @Override public void unparseIntervalOperandsBasedFunctions(SqlWriter writer,
+      SqlCall call, int leftPrec, int rightPrec) {
+    final SqlWriter.Frame frame = writer.startFunCall(call.getOperator().toString());
+    call.operand(0).unparse(writer, leftPrec, rightPrec);
+    writer.sep(",");
+    switch (call.operand(1).getKind()) {
+    case LITERAL:
+      call.operand(1).unparse(writer, leftPrec, rightPrec);
+      break;
+    default:
+      SqlBasicCall secondOperand = call.operand(1);
```

Comment:
Give a more meaningful variable name like intervalOperand instead of second operand